src/tools/pub/assets-and-transformers.md

```diff
@@ -1,397 +0,0 @@
----
```

Comment:
As much as I love deleting code, do we really want to completely delete this? What about people who'll want to use Dart 1 for a few months longer? Can't we just paste a huge "This is Dart 1" banner over this?

Reply:
I believe we want to delete as much as possible, to have a clean slate for Dart 2. Is that right, @kevmoo and @dgrove? We will, as I mentioned in another comment, keep https://v1-dartlang-org.firebaseapp.com/ (the 1.x fork of site-www) up for a while.

Reply:
I'm also nervous about just dropping this content – I wonder if having a place-holder page is better – which links to the v1 docs or the the v2 content?

Reply:
I've created https://kw-www-dartlang-1.firebaseapp.com/tools/pub/obsolete to do this. What do you think?